Bryn Mawr Classical Review.js

```diff
@@ -9,7 +9,7 @@
 	"inRepository": true,
 	"translatorType": 4,
 	"browserSupport": "gcsibv",
-	"lastUpdated": "2015-06-02 10:52:41"
+	"lastUpdated": "2016-04-07 21:36:13"
 }
 
```

Comment:
AFAIS the contributors to this translators are the usual persons and John. Therefore, we can also add the AGPL license block here. CC @adam3smith @aurimasv Right?